playground/recipe_opstack.go

```diff
@@ -175,6 +182,21 @@ func (o *OpRecipe) Apply(ctx *ExContext, artifacts *Artifacts) *Manifest {
 		MaxChannelDuration: o.batcherMaxChannelDuration,
 	})
 
+	if o.enableChainMonitor {
+		l2BlockTime := fmt.Sprintf("%ds", o.blockTime)
+
+		svcManager.AddService("chain-monitor", &ChainMonitor{
+			L1RPC:            Connect("el", "http"),
```

Comment:
If possible and it is always the same, I prefer to use only the service name (`el`) and let the component call Connect itself.